web/src/stream_create.js

```diff
@@ -78,6 +82,12 @@ class StreamNameError {
         $("#create_stream_name").trigger("focus").trigger("select");
     }
 
+    rename_archived_stream(stream_name) {
+        $("#archived_stream_rename").text($t({defaultMessage: "Rename archived stream"}));
+        $("#archived_stream_rename").attr("data-stream-name", stream_name);
```

Comment:
Can we use a `data-stream-id` instead? I guess this may be a case where we really do need the name; we just try to avoid working with mutable stream names where possible.

Reply:
Yes we can.
I updated the function.